.teamcity/Gradle_Check/configurations/Gradleception.kt

```diff
@@ -38,7 +38,7 @@ class Gradleception(model: CIBuildModel, stage: Stage) : BaseGradleBuildType(mod
         }
         localGradle {
             name = "QUICKCHECK_WITH_GRADLE_BUILT_BY_GRADLE"
-            tasks = "clean sanityCheck test"
+            tasks = "clean sanityCheck test distributionsIntegTests:quickTest"
```

Comment:
This will make this build (which runs overtime) even longer. But I guess it's worth it. You could use `:distributionsIntegTests:embeddedIntegTest` here instead then it is a bit clearer what we are running. (The project has no "unit" tests, and `test` is called for all projects already anyway.)

Reply:
Got this:

```
java.lang.RuntimeException: Running tests with a Gradle distribution in embedded mode is no longer supported.
	at org.gradle.integtests.fixtures.executer.GradleContextualExecuter.configureExecuter(GradleContextualExecuter.java:117)
	at org.gradle.integtests.fixtures.executer.AbstractDelegatingGradleExecuter.assertCanExecute(AbstractDelegatingGradleExecuter.java:39)
	at org.gradle.integtests.fixtures.executer.AbstractGradleExecuter.beforeBuildSetup(AbstractGradleExecuter.java:1138)
	at org.gradle.integtests.fixtures.executer.AbstractGradleExecuter.run(AbstractGradleExecuter.java:1095)
	at org.gradle.DistributionIntegrationSpec.checkMinimalContents(DistributionIntegrationSpec.groovy:152)
	at org.gradle.AllDistributionIntegrationSpec.allZipContents(AllDistributionIntegrationSpec.groovy:45)
Caused by: java.lang.AssertionError: expected null, but was:<bin/gradle>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotNull(Assert.java:756)
	at org.junit.Assert.assertNull(Assert.java:738)
	at org.junit.Assert.assertNull(Assert.java:748)
	at org.gradle.integtests.fixtures.executer.InProcessGradleExecuter.assertCanExecute(InProcessGradleExecuter.java:393)
	at org.gradle.integtests.fixtures.executer.GradleContextualExecuter.configureExecuter(GradleContextualExecuter.java:114)
	... 5 more
```